storybookjs/storybook
# UI: Fix target id in searchfield label

Closes #23463

## What I did

The for attribute value was not set to match the hardcoded value used for the search input id. This is invalid HTML and creates an error issue in chrome dev tools "Issues" tab.

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Open Storybook in your Chrome browser.
3. Observe error in "issues" tab of Chrome dev tools.

## Checklist

- NA Make sure to add/update documentation regarding your changes
- NA If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
